src/components/Patient/LinkDepartmentsSheet.tsx

```diff
@@ -292,7 +311,7 @@ export default function LinkDepartmentsSheet({
           </Button>
         )}
       </SheetTrigger>
-      <SheetContent>
+      <SheetContent className="!w-[50vw] !max-w-none min-h-[50vh] overflow-auto">
```

Comment:
co-pilot will say otherwise, but latest tailwind adds ! at the end.

https://tailwindcss.com/docs/styling-with-utility-classes#using-the-important-modifier